content/ios/asset.md

```diff
@@ -1,5 +1,9 @@
 ### Uploading and associating an asset to a record
 
+You can make use of `Asset` to store file references such as images and videos on the database. An asset can only be saved with a record but not as a standalone upload. Skygear automatically uploads the files to a server that you specify, like Amazon S3.
+
+For example, you want to allow users to upload an image as an `attachment` to his `note`. Once the user has selected the image to upload, you can save it by:
```

Comment:
the description does not match with the code. How about we follow the code to use profile pic as the example? We will need to modify the js part too